params/denomination.go

```diff
@@ -25,3 +27,7 @@ const (
 	GWei  = 1e9
 	Ether = 1e18
 )
+
```

Comment:
This doesn't seem to be used.

Reply:
Automatically added by vscode, and I can't remove it without using a different editor